Automattic/wp-calypso
# Signup: Remove unused `upgrade` flow / `PaidPlansWithSkip`

Fixes #3734. The `upgrade` flow is no longer used, so we can get rid of it, as well as the `PaidPlansWithSkip` component, which was only used for that flow.

**Testing**
*the non-existent `upgrade` flow*
- Visit `/start/upgrade`
- Assert that you are redirected to `/start/themes`

*the main flow*
- Visit `/start`
- Proceed to the `plans` step
- Assert that you see three plans.
- Select one of the plans and complete the signup.
- Assert that you are redirected to `yoursite.wordpress.com` or checkout, depending on whether you chose a paid plan.
